plugin/src/main/resources/io/jenkins/plugins/analysis/warnings/groovy/ParserConfiguration/config.properties

```diff
@@ -1 +1,2 @@
 title.description=Create new parsers for the warnings plug-in based on a regular expression and a Groovy script.
+consoleLogScanningPermitted.description=Allow these parsers to act upon the build console.
```

Comment:
Jenkins's security team mentioned that it would be helpful to make the text from your help file visible here as well (`help-canScanConsoleLog.html`). At least that checking the box can introduce a (potential) security risk.

Reply:
I'd prefer not to clutter the config page with a duplication of the help info.
...but I can do better than that - I'll add a doCheck method that'll show a warning when the checkbox is ticked that tells people to go read the help.
I suspect that a yellow warning will get more attention than static text.